source/include/CEntryPoints.h

```diff
@@ -41,13 +41,17 @@ VIREO_EXPORT void EggShell_WriteValueString(TypeManagerRef tm, const char* viNam
                                             const char* format, const char* value);
 VIREO_EXPORT const char* EggShell_ReadValueString(TypeManagerRef tm, const char* viName, const char* eltName,
                                                   const char* format);
-VIREO_EXPORT EggShellResult EggShell_GetArrayMetadata(TypeManagerRef tm, const char* viName, const char* eltName,
-    char** arrayTypeName, Int32* arrayRank, unsigned char** arrayBegin);
+VIREO_EXPORT EggShellResult EggShell_GetArrayPointer(TypeManagerRef tm,
+        const char* viName, const char* eltName, void** arrayPointer);
 VIREO_EXPORT Int32 EggShell_GetArrayDimLength(TypeManagerRef tm, const char* viName, const char* eltName, Int32 dim);
 VIREO_EXPORT Int32 EggShell_ResizeArray(TypeManagerRef tm, const char* viName, const char* eltName,
                                         Int32 rank, Int32* newLengths);
 VIREO_EXPORT void* Data_GetStringBegin(StringRef stringObject);
 VIREO_EXPORT Int32 Data_GetStringLength(StringRef stringObject);
+VIREO_EXPORT EggShellResult Data_GetArrayMetadata(TypeManagerRef tm,
```

Comment:
I wonder if this function is used in the JS control layer of the Web VI. Please run an LVjs CI with the changes and report the results.